packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -322,6 +322,17 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
               )}
             </span>
           )}
+          {!!onSearch && (
+            <Button
```

Comment:
This should be a child of the `<InputGroup>`, not `<div className={css(styles.searchInputBar)}>`